src/shogun/evaluation/Calibration.cpp

```diff
@@ -0,0 +1,282 @@
+/*
+ * Copyright (c) The Shogun Machine Learning Toolbox
+ * Written (w) 2012 - 2013 Heiko Strathmann
```

Comment:
you might want to add you own name here :)

Reply:
Okay :)